common.blocks/issue/issue.js

```diff
@@ -36,13 +38,21 @@ provide(bemDom.declBlock(this.name, {
         var footer = this._elem('footer');
 
         event.preventDefault();
-        button.setMod('disabled');
 
-        request(button.params.number + '/comments')
-            .then(function(response) { return response.text(); })
-            .then(function(data) {
-                bemDom.append(footer.domElem, data);
-            });
+        if (!this.isCommentsOpened) {
+            this.isCommentsOpened = true;
```

Comment:
you need to do a private function called like _toggleCommentsVisibility
in that function you should check present state of this._isCommentsVisible and invert it